autoload/neomake/makers/ft/php.vim

```diff
@@ -27,9 +27,8 @@ endfunction
 function! neomake#makers#ft#php#phpcs() abort
     let l:args = ['--report=csv']
 
-    "Add standard argument if one is set.
-    if exists('g:neomake_php_phpcs_args_standard')
-        call add(l:args, '--standard=' . expand(g:neomake_php_phpcs_args_standard))
+    if exists('g:neomake_php_phpcs_args_post')
+        call add(l:args, , g:neomake_php_phpmd_args_post))
```

Comment:
The shell maker tests with their argument validation should give me a rough guide to testing.

Been thinking about this more and looking back on the way Syntastic handles it (see below) and now I am rethinking the approach.

```viml
let g:syntastic_php_phpcs_args = "--standard=PSR2"
let g:syntastic_php_phpmd_post_args = "cleancode,codesize,controversial,design,naming,unusedcode"
```
Would a solution be that the user can put anything into args but the maker adds or overrides as necessary in order for it to work with the error format?